master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java

```diff
@@ -320,6 +321,7 @@ public void updateRegisterWorkerMeta(
             fetchPort,
             replicatePort,
             internalPort,
+            version,
```

Comment:
Due to it only update the `workersMap` if absent, so if we restart the celeborn worker, it won't update the WorkerInfo.

https://github.com/apache/celeborn/blob/44d772df75e17cb485d1e4aca086e5fc5b2bd2d6/master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java#L335

Reply:
fixed